src/components/09-buttons/buttons.config.yml

```diff
@@ -1,8 +1,47 @@
+title: Buttons
 status: "ready"
+context:
+    label: "Primary"
```

Comment:
I think in the context of a button we should consider updating `label` to `content`.